src/sdk/pynni/nni/msg_dispatcher.py

```diff
@@ -114,7 +114,7 @@ def handle_import_data(self, data):
         data: a list of dictionaries, each of which has at least two keys, 'parameter' and 'value'
         """
         for entry in data:
-            entry['value'] = json_tricks.loads(entry['value'])
+            entry['value'] = json_tricks.loads(str(entry['value']))
```

Comment:
what is the type of `entry['value']`? a `dict` object?

Reply:
In this simple case like `[{"parameter": {...}, "value": 0.03}]`, it is `<class 'float'>`

Reply:
if it is float, why we invoke `json_tricks.loads`?

Reply:
maybe in some complex scene, user will set value as other types? like numpy array? This should invoke `json_tricks.loads` to converted back.

Reply:
i think `entry['value']` should be a string, otherwise, there is a bug somewhere else.